src/mumble/AudioInput.cpp

```diff
@@ -384,6 +384,14 @@ void AudioInput::addMic(const void *data, unsigned int nsamp) {
 					if (qlEchoFrames.isEmpty()) {
 						iJitterSeq = 0;
 						iMinBuffered = 1000;
+					} else if (qlEchoFrames.count() >= 100) {
```

Comment:
So is this like cutoff in case the drift handling in the else case is insufficient? The else case (please correct me if I'm wrong) seems to be able to drop 10ms for every second of audio that was played without getting below 10ms of buffer. Could you document the relationship? In general I find iJitterSeq and iMinBuffered to be non intuitive.